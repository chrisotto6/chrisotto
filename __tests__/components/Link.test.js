import { render } from '@testing-library/react'
import CustomLink from '@/components/Link'

it('renders slash links unchanged', () => {
  const { container } = render(<CustomLink href="/test">{'Test'}</CustomLink>)
  expect(container).toMatchSnapshot()
})

it('renders anchor links unchanged', () => {
  const { container } = render(<CustomLink href="#test">{'Test'}</CustomLink>)
  expect(container).toMatchSnapshot()
})

it('renders real links unchanged', () => {
  const { container } = render(<CustomLink href="https://www.google.com">{'Test'}</CustomLink>)
  expect(container).toMatchSnapshot()
})
