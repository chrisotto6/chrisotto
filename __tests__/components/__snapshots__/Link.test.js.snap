// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders anchor links unchanged 1`] = `
<div>
  <a
    href="#test"
  >
    Test
  </a>
</div>
`;

exports[`renders real links unchanged 1`] = `
<div>
  <a
    href="https://www.google.com"
    rel="noopener noreferrer"
    target="_blank"
  >
    Test
  </a>
</div>
`;

exports[`renders slash links unchanged 1`] = `
<div>
  <a
    href="/test"
  >
    Test
  </a>
</div>
`;
